test/lang.test.js

```diff
@@ -23,39 +25,39 @@ describe('Localisation', function() {
 			locale = {};
 		}
 
-		describe(`# Language: ${lang.toUpperCase()}`, function() {
+		describe(`# Language: ${lang.toUpperCase()}`, function () {
```

Comment:
Your linter adds a space between `function` and `()` its parentheses. Could you remove these extra spaces?

Reply:
I have pushed a fix myself. Is it OK for you?

Reply:
Oh, that's ok. I just ran a format over the file. VS Code wants the code to look like this 😅

Reply:
This method should find almost all calls. There are some instances it won't find because the key is build dynamically. But those are some small edge cases.